test/test_apps/triangle/triangle.cpp

```diff
@@ -201,61 +214,61 @@ void create_graphics_pipeline(gfxrecon::test::Init& init, RenderData& data) {
     pipeline_info.pMultisampleState = &multisampling;
     pipeline_info.pColorBlendState = &color_blending;
     pipeline_info.pDynamicState = &dynamic_info;
-    pipeline_info.layout = data.pipeline_layout;
-    pipeline_info.renderPass = data.render_pass;
+    pipeline_info.layout = this->pipeline_layout;
+    pipeline_info.renderPass = this->render_pass;
     pipeline_info.subpass = 0;
     pipeline_info.basePipelineHandle = VK_NULL_HANDLE;
 
-    result = init.disp.createGraphicsPipelines(VK_NULL_HANDLE, 1, &pipeline_info, nullptr, &data.graphics_pipeline);
+    result = init.disp.createGraphicsPipelines(VK_NULL_HANDLE, 1, &pipeline_info, nullptr, &this->graphics_pipeline);
     VERIFY_VK_RESULT("failed to create graphics pipeline", result);
 
     init.disp.destroyShaderModule(frag_module, nullptr);
     init.disp.destroyShaderModule(vert_module, nullptr);
 }
 
-void create_framebuffers(gfxrecon::test::Init const& init, RenderData& data) {
-    data.framebuffers.resize(init.swapchain_image_views.size());
+void Triangle::create_framebuffers() {
+    this->framebuffers.resize(init.swapchain_image_views.size());
 
     for (size_t i = 0; i < init.swapchain_image_views.size(); i++) {
         VkImageView attachments[] = { init.swapchain_image_views[i] };
 
         VkFramebufferCreateInfo framebuffer_info = {};
         framebuffer_info.sType = VK_STRUCTURE_TYPE_FRAMEBUFFER_CREATE_INFO;
-        framebuffer_info.renderPass = data.render_pass;
+        framebuffer_info.renderPass = this->render_pass;
         framebuffer_info.attachmentCount = 1;
         framebuffer_info.pAttachments = attachments;
         framebuffer_info.width = init.swapchain.extent.width;
         framebuffer_info.height = init.swapchain.extent.height;
         framebuffer_info.layers = 1;
 
-        auto result = init.disp.createFramebuffer(&framebuffer_info, nullptr, &data.framebuffers[i]);
+        auto result = init.disp.createFramebuffer(&framebuffer_info, nullptr, &this->framebuffers[i]);
         VERIFY_VK_RESULT("failed to create framebuffer", result);
     }
 }
 
-void create_command_buffers(gfxrecon::test::Init& init, RenderData& data) {
-    data.command_buffers.resize(data.framebuffers.size());
+void Triangle::create_command_buffers() {
+    this->command_buffers.resize(this->framebuffers.size());
```

Comment:
Do you really need to use `this->` everywhere here?

Reply:
Fixed